service.go

```diff
@@ -818,9 +818,17 @@ func handleServiceChannelStatus(dc *downstreamConn, params []string) error {
 	n := 0
 
 	sendNetwork := func(net *network) {
+		var channels []*Channel
 		for _, entry := range net.channels.innerMap {
-			ch := entry.value.(*Channel)
+			channels = append(channels, entry.value.(*Channel))
+		}
+
+		sort.SliceStable(channels, func(i, j int) bool {
```

Comment:
Any reason why a stable sort is used here? Can't we just use `sort.Slice`?

Reply:
sort.Slice is the correct choice. I used sort.SliceStable thinking it would prevent two channels on different networks from swapping positions except 1) networks are processed individually and 2) the slice is created every time the channel status command is called, so SliceStable has no benefit.